reports/spec2.tex

```diff
@@ -134,7 +134,7 @@ \subsection{Checking the Specification}
 on top of these graphs.
 
 
-To further evaluate \SpecTwo{}, we fix the block graph --- this way the solver
+To further evaluate \SpecTwo{}, we fix the block graph\footnote{We consider forests, rather than just trees, because the specification allows for that. Specifically, forests can occur when only some of the blocks of a given chain are received by a node. In the specification, the set of blocks received corresponds the map \texttt{CommonNodeState.view\_blocks}.} --- this way the solver
```

Comment:
```suggestion
To further evaluate \SpecTwo{}, we fix the block graph\footnote{We consider forests, rather than just trees, because the specification allows for that. Specifically, forests can occur when only some of the blocks of a given chain are received by a node. In the specification, the set of blocks received corresponds to the map \texttt{CommonNodeState.view\_blocks}.} --- this way the solver
```